source/api/image/3.0/index.md

```diff
@@ -424,15 +427,15 @@ It is possible to request the same image using different combinations of paramet
   * Caching becomes significantly more efficient, both client and server side, when the URIs used are the same between systems and sessions.
   * Response times can be improved by avoiding redirects from a requested non-canonical URI syntax to the canonical syntax by using the canonical form directly.
 
-In order to support the above requirements, clients _SHOULD_ construct the image request URIs using the following canonical parameter values where possible.  Image servers _MAY_ redirect the client to the canonical URI from a non-canonical equivalent.
+In order to support the above requirements, clients _SHOULD_ construct image request URIs using the following canonical parameter values where possible.  Image servers _MAY_ redirect the client to the canonical URI from a non-canonical equivalent.
 
 | Parameter | Canonical value |
 | --------- | --------------- |
-| region    | "full" if the whole image is requested<br/>otherwise the `x,y,w,h` syntax. |
+| region    | "full" if the full image is requested<br/>otherwise the `x,y,w,h` syntax. |
```

Comment:
Same as https://github.com/IIIF/api/pull/1462/files#r170224974

Reply:
As in https://github.com/IIIF/api/pull/1462/files#r170224974, I think it should be **full image** which we define as the starting point for the transformations